build-config.ts

```diff
@@ -9,6 +9,8 @@ export default {
   inputDir: process.env.COMMERCE_SDK_INPUT_DIR || `${__dirname}/apis`,
   updateApiDir: process.env.COMMERCE_SDK_UPDATE_API_DIR || `${__dirname}/updateApiTmpDir`,
   renderDir: process.env.COMMERCE_SDK_RENDER_DIR || `${__dirname}/packages/generator/renderedTemplates`,
+  
+  // TODO: This this should become 'group by' and provide a method to pickout the metadata you want to group by
```

Comment:
Please add a story for the todo.

Reply:
There already is one.  I am pretty sure @bredmond-sf wrote a story for this in his migration plan. I know I have discussed it with him.  Was adding this as a kind of marker to start talking about what that would look like.

Reply:
This also kind of serves as instructions for what to do with that line when you work on it.